client/hosting/server-settings/components/defensive-mode-card/index.tsx

```diff
@@ -100,16 +102,52 @@ export default function DefensiveModeCard( { disabled }: DefensiveModeCardProps
 						</HostingCardDescription>
 					</div>
 
-					<Button
-						className="defensive-mode-card__button"
-						busy={ isEnabling }
-						disabled={ disabled || isEnabling }
-						onClick={ () => {
-							mutate( { active: false } );
-						} }
-					>
-						{ translate( 'Disable defensive mode' ) }
-					</Button>
+					{ ! defensiveModeData.enabled_by_a11n && (
+						<Button
+							className="defensive-mode-card__button"
+							busy={ isEnabling }
+							disabled={ disabled || isEnabling }
+							onClick={ () => {
+								mutate( { active: false } );
+							} }
+						>
+							{ translate( 'Disable defensive mode' ) }
+						</Button>
+					) }
+
+					{ defensiveModeData.enabled_by_a11n && (
+						<div className="defensive-mode-card__a11n-description">
+							<div>
+								<Icon icon={ info } size={ 24 } />
+							</div>
+
+							<div className="defensive-mode-card__a11n-text">
+								<p>
+									<strong>
+										{ translate(
+											'Defensive mode was enabled on your behalf to protect your site.'
+										) }
+									</strong>
+								</p>
+								<p>
+									{ translate(
+										'Please {{a}}contact support{{/a}} if you need to disable defensive mode.',
+										{
+											components: {
+												a: (
+													<a
+														href="https://wordpress.com/help/contact"
+														target="_blank"
+														rel="noopener noreferrer"
+													/>
```

Comment:
I would ideally like to open a support chat directly and bypass Odie. TBD. See p1727267111981439-slack-C07D72S1135